docs/guides/measure-qubits.ipynb

```diff
@@ -191,6 +198,38 @@
     "qc.measure_active()  # Measure qubits that are not idle, i.e., qubits 0 and 2."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "b1c5f875-6dfb-48e9-83f4-b25ba516af83",
+   "metadata": {},
+   "source": [
+    "### `MidCircuitMeasure` method\n",
+    "\n",
+    "\n",
+    "\n",
+    "Use `MidCircuitMeasure` (**NEED LINK**) to apply a mid-circuit measurement.  While you can use `QuantumCircuit.measure` for a mid-circuit measurement, `MidCircuitMeasure` is designed specifically for mid-circuit measurements, and is therefore typically a better choice.  For example, it adds less overhead to your circuit than when using QuantumCircuit.measure`."
```

Comment:
Are there other reasons to choose MidCircuitMeasure, or is reduced overhead the only one worth mentioning? If that's the big one, maybe reword to "While you can use `QuantumCircuit.measure` for a mid-circuit measurement, `MidCircuitMeasure` adds less overhead to your circuit, and is therefore typically a better choice." (This avoids the seemingly-obvious statement that MidCircuitMeasure is designed specifically for mid-circuit measurements)